bom/pom.xml

```diff
@@ -65,7 +65,7 @@
 
         <!-- Neo4j -->
         <neo4jVersion>3.3.3</neo4jVersion>
-        <neo4jJavaDriverVersion>1.5.0</neo4jJavaDriverVersion>
+        <neo4jJavaDriverVersion>1.5.1</neo4jJavaDriverVersion>
```

Comment:
Good catch, looks like I missed it in the last upgrade. Thanks.

Will wait for CI then merge.